RockWeb/Themes/NewSpring/assets/Lava/UI/footer-network.lava

```diff
@@ -0,0 +1,58 @@
+<style>
+.fa-facebook-square, .fa-twitter-square, .fa-instagram { transition: all .2s ease; }
+.fa-facebook-square:hover { color: #4467ca; }
+.fa-twitter-square:hover { color: #1DA1F2; }
+.fa-instagram:hover { color: #FDD362 }
+</style>
+
+{% assign orgEmail = 'Global' | Attribute:'OrganizationEmailNetwork' %}
+{% assign orgPhone = 'Global' | Attribute:'OrganizationPhoneNetwork' %}
+
+
+<footer class="position-relative shell soft-double xs-soft-double-ends bg-gray-dark rounded shadowed">
+    <a href="#content" data-scroll title="Scroll to Top">
+        <img src="//dg0ddngxdz549.cloudfront.net/newspring/icons/newspring_mark.png" alt="NewSpring Network" class="position-absolute bottom-zero right-zero" style="max-width: 50px;">
+    </a>
+    <div class="row sans-serif strong text-decoration-none">
+        <div class="col-xs-12 col-sm-6 col-md-3 xs-text-center push-bottom xs-push-half-bottom">
+            <h2 class="h5 text-white push-half-bottom">NewSpring Network</h2>
+            <span class="text-white">
+                <p class="flush-bottom xs-push-half-bottom">
+                    <a href="tel:+1{{ orgPhone | Remove:'(' | Remove:')' | Remove:'-' | Remove:' ' }}">{{ orgPhone }}</a><br>
+                    <a href="mailto:{{ orgEmail }}">{{ orgEmail }}</a>
+                </p>
+            </span>
+        </div><div class="col-xs-12 col-sm-6 col-md-3 push-bottom xs-push-half-bottom">
+            <h2 class="h5 text-white xs-text-center push-half-bottom">NewSpring Network</h2>
+            <ul class="list-unstyled xs-text-center flush-bottom xs-push-half-bottom">
+                <li><a href="/">Home</a></li>
+                <li><a href="/vision">About NewSpring Network</a></li>
+                <li><a href="/questions">FAQs</a></li>
+                <li><a href="/resources">Resources</a></li>
+                <li><a href="/connect">Connect With Us</a></li>
+            </ul>
+        </div><div class="col-xs-12 col-sm-6 col-md-3 push-bottom xs-push-half-bottom">
+            <h2 class="h5 text-white xs-text-center push-half-bottom">Resources</h2>
+            <ul class="list-unstyled xs-text-center flush-bottom xs-push-half-bottom">
+                <li><a href="/resources/kids">Kids Series</a></li>
+                <li><a href="/resources/students">Student Series</a></li>
+                <li><a href="/resources/series">Sermon Series</a></li>
+                <li><a href="/resources/volunteers">Volunteers</a></li>
```

Comment:
This is missing "Music". Is that on purpose?

Reply:
@briankalwat I know this PR is already closed, but this didn't get addressed. Was removing "Music" on purpose?